src/main/resources/assets/js/app/view/context/WidgetsSelectionRow.ts

```diff
@@ -118,3 +124,50 @@ export class WidgetViewOption {
     }
 
 }
+
+export class WidgetViewer extends NamesAndIconViewer<WidgetViewOption> {
+
+    constructor() {
+        super('widget-viewer');
+    }
+
+    doLayout(object: WidgetViewOption) {
+        super.doLayout(object);
+
+        const view = this.getNamesAndIconView();
+        if (object && object.getWidgetView() && view) {
+            const widgetClass = object.getWidgetView().getWidgetKey() != null ? 'external-widget' : 'internal-widget';
+            view.removeClass('external-widget internal-widget');
+            view.addClass(widgetClass);
+        }
+    }
+
+    resolveDisplayName(object: WidgetViewOption): string {
+        return object.getWidgetView().getWidgetName();
+    }
+
+    resolveSubName(object: WidgetViewOption): string {
+        const name = object.getWidgetView().getWidgetName();
+        const description = object.getWidgetView().getWidgetDescription() || 'No description';
+        switch (name) {
+        case 'Properties': return 'Properties of the content';
+        case 'Version history': return 'History of the content\'s changes';
+        case 'Dependencies': return 'Dependencies tree of the content';
+        default:
+            return description;
+        }
+    }
+
+    resolveIconUrl(object: WidgetViewOption): string {
+        return object.getWidgetView().getWidgetIconUrl();
+    }
+
+    resolveIconClass(object: WidgetViewOption): string {
+        const name = object.getWidgetView().getWidgetName();
+        switch (name) {
+        case 'Properties': return 'icon-info';
```

Comment:
Let's change the icon of this widget to the same icon that we used to have for the detail panel toggler (``icon-list``). Also remove the icon-info since we no longer need it.